docs/decision_record.md

```diff
@@ -17,3 +17,22 @@ Should a wider GA4GH standard appear from [TASC issue 5](https://github.com/ga4g
 ### Known limitations
 
 - There have been a handful of reports of old sequences with disallowed characters in the sequence name rows (`>`) of FASTA files, particularly from the microbiome community. These sequence collections would have to be changed to include only SAM-compatible ASCII characters, which could restrict usage of the sequence collections protocols and delay uptake.
+
+## 2021-09-22 - Controlled vocabulary for the supported fields
+
+### Decision
+
+The attributes allowed to be used in the sequence collection digest construction are: 
```

Comment:
I'm not sure if 'allowed' is the right word here -- since we allow everything, really. I think it's more like, these are the controlled vocabulary terms for the basic, universal seqcol spec....

Maybe:

"The initial set of controlled vocabulary terms for sequence collection attributes included in the digest construction are: "

And then I think I'd also add to this ADR the decision that additional custom attributes are allowed.